vendor/ocaml-4.12/dune

```diff
@@ -0,0 +1,12 @@
+(library
+ (name ocaml_412)
+ (modules_without_implementation let_binding)
+ (flags
+  (:standard -w -9))
+ (libraries compiler-libs.common menhirLib))
+
+(ocamllex lexer)
+
+(menhir
```

Comment:
The `MenhirBasics.Error` thing is not supposed to happen. Here are the flags that are used in the compiler (I removed `--require-aliases`):

```
(menhir
 (flags
  --lalr
  --explain
  --dump
  --strict
  -lg
  1
  -la
  1
  --unused-token
  COMMENT
  --unused-token
  DOCSTRING
  --unused-token
  EOL
  --unused-token
  GREATERRBRACKET
  --fixed-exception
  --table
  --strategy
  simplified)
 (modules parser))
```

Reply:
Great, thanks! the CI is alright now, the outputs of the tests are okay since the code printing the error messages is vendored anyway